From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Reilly Brogan <reilly@reillybrogan.com>
Date: Thu, 21 Mar 2024 14:28:37 -0500
Subject: [PATCH] Revert "cmake: Fix man page generation"

This reverts commit 966ad9777ee27b4676eecf5832c9d066b2c74610.

This breaks docbook-xslt detection on Solus
---
 cmake/FindDocBookXSL.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/FindDocBookXSL.cmake b/cmake/FindDocBookXSL.cmake
index fb905d6a4..9192ef9b3 100644
--- a/cmake/FindDocBookXSL.cmake
+++ b/cmake/FindDocBookXSL.cmake
@@ -30,7 +30,7 @@ if (NOT DOCBOOKXSL_DIR)
    set (STYLESHEET_PATH_LIST)
    foreach (STYLESHEET_PREFIX_ITER ${CMAKE_SYSTEM_PREFIX_PATH})
       file(GLOB STYLESHEET_SUFFIX_ITER RELATIVE ${STYLESHEET_PREFIX_ITER}
-           ${STYLESHEET_PREFIX_ITER}/share/xml/docbook/xsl-stylesheets-*-nons
+           ${STYLESHEET_PREFIX_ITER}/share/xml/docbook/xsl-stylesheets-*
       )
       if (STYLESHEET_SUFFIX_ITER)
          list (APPEND STYLESHEET_PATH_LIST ${STYLESHEET_SUFFIX_ITER})
